tests/src/Unit/Entity/MessageTemplateTest.php

```diff
@@ -108,4 +122,96 @@ public function testIsLocked() {
     $this->assertFalse($this->messageTemplate->isLocked());
   }
 
+  /**
+   * Tests the getText method.
+   *
+   * @covers ::getText
+   */
+  public function testGetText() {
```

Comment:
wow, it's really nice to see this as unit test!

I think it's worth spliting it to smaller methods, as right now i kind of checks everything 😄